go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/cockroachdb/apd v1.1.0 // indirect
 	github.com/cockroachdb/cockroach-go v0.0.0-20181001143604-e0a95dfd547c
 	github.com/coreos/go-semver v0.2.0
+	github.com/coreos/go-systemd v0.0.0-20181012123002-c6f51f82210d // indirect
```

Comment:
These additions go away when I `go mod tidy`.

Reply:
For me only the kr/pty one does, but not the others.  Feels like a lost cause to try to keep it one way or the other, esp with many of using IDEs that run go commands in the background that may impact go.mod.

Reply:
Yeah it's very erratic, and changes regularly it seems based on the version of the go tool you run.